src/modules/tpm/src/lib/Tpm.h

```diff
@@ -1,9 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#pragma once
```

Comment:
Is there any potential for incompatibility with any of the compilers we support? My understanding is this will fail silently if a compiler does not support `#pragma once`.

Reply:
Do not use pragma once for a C++ header.

Reply:
Great point, addressed this in the latest iteration.

Reply:
There is no new iteration. Please push...